customer-service/omnichannel/smart-assist-bot/README.md

```diff
@@ -1,14 +1,158 @@
+3) Update the Messaging endpoint in Bot channels registration<br />
+4) Enable Teams channel<br />
+5) Test your Bot online<br />
+
+**Update appsettings.json** <br />
+
+Go to [Appsettings.json](https://github.com/microsoft/Dynamics365-Apps-Samples/blob/master/customer-service/omnichannel/smart-assist-bot/SmartAssistBot/appsettings.json) and update LUIS, Bot and CDS settings following the steps mentioned below.
+
+1) **LUIS Settings**
+
+    Follow the instructions mentioned in topic [Add natural language understanding to your bot](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-v4-luis?view=azure-bot-service-4.0&tabs=csharp) to add LUIS to your bot. Once you have created a LUIS app, see [this section](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-v4-luis?view=azure-bot-service-4.0&tabs=csharp#retrieve-application-information-from-the-luisai-portal) to get `LuisAppId`, `LuisAPIKey` and `LuiAPIHostName`
+
+
+    Sample screenshots
+    ![](screenshots/LUIS1.png)
+
+    ![](screenshots/LUISAppID.png)
+
+    You can create intents according to your scenario. Here for this sample code to work please create intent for appointment.
+
+    ![](screenshots/intent1.png)
+
+    ![](screenshots/intent2.png)
+
+    **LUIS** app settings can be left blank if you are interested only in KB search
+    
+        // LUIS connection settings
+        "LuisAPIHostName": "westus.api.cognitive.microsoft.com",
+        "LuisAPIKey": "", // LUIS APP Key
+        "LuisAppId": "", // LUIS APP ID
+
+2) **BOT Settings**
+
+    Register your bot with Azure Bot Service and obtain the Microsoft App ID and a Client secret like mentioned below<br />
+
+    a. Create registration resource following [this documentation](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-3.0#create-a-registration-resource)<br />
+
+    ![](screenshots/Botsettings_bcr.png)
+
+    b. Go to the resource which is just created and select settings from left pane and then click on Manage like shown below in the screenshot.<br />
+
+    ![](screenshots/BotAppId.png)
+
+    c. Generate Client secret like mentioned [here](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-3.0#get-registration-password) and copy it separately. This will be your _MicrosoftAppPassword_<br />
+
+    ![](screenshots/AppPassword.png)
+
+    d. Now click on **Overview** from left pane and copy the _Application(client) ID_.Update both the Microsoft App ID and secret in appsettings.json
+
+    ![](screenshots/BotAppId2.png)
+
+
+    ```
+    // Bot settings
+    "MicrosoftAppId": "<Microsoft App Id as copied in step 2d>",
+    "MicrosoftAppPassword": "<Microsoft App password as copied in step 2c>",
+    "REMOTEDEBUGGINGVERSION": "15.0.28307.222",
+    "ScmType": "None"```
+
+3) **CDS settings**
+
+    a. Login to Azure portal with CDS credentials<br />
```

Comment:
Login withe tenant credentials?

Reply:
Tested with non admin user and it worked fine.